tutorials/colors/colors.py

```diff
@@ -5,59 +5,105 @@
 
 Matplotlib recognizes the following formats to specify a color:
 
-* an RGB or RGBA (red, green, blue, alpha) tuple of float values in closed
-  interval ``[0, 1]`` (e.g., ``(0.1, 0.2, 0.5)`` or ``(0.1, 0.2, 0.5, 0.3)``);
-* a hex RGB or RGBA string (e.g., ``'#0f0f0f'`` or ``'#0f0f0f80'``;
-  case-insensitive);
-* a shorthand hex RGB or RGBA string, equivalent to the hex RGB or RGBA
-  string obtained by duplicating each character, (e.g., ``'#abc'``, equivalent
-  to ``'#aabbcc'``, or ``'#abcd'``, equivalent to ``'#aabbccdd'``;
-  case-insensitive);
-* a string representation of a float value in ``[0, 1]`` inclusive for gray
-  level (e.g., ``'0.8'`` for light gray);
-* one of the characters ``{'b', 'g', 'r', 'c', 'm', 'y', 'k', 'w'}``, which
-  are short-hand notations for shades of blue, green, red, cyan, magenta,
-  yellow, black, and white. Note that the colors ``'g', 'c', 'm', 'y'`` do not
-  coincide with the X11/CSS4 colors. Their particular shades were chosen for
-  better visibility of colored lines against typical backgrounds.
-* a X11/CSS4 color name (case-insensitive);
-* a name from the `xkcd color survey`_, prefixed with ``'xkcd:'`` (e.g.,
-  ``'xkcd:sky blue'``; case insensitive);
-* one of the Tableau Colors from the 'T10' categorical palette (the default
-  color cycle): ``{'tab:blue', 'tab:orange', 'tab:green', 'tab:red',
-  'tab:purple', 'tab:brown', 'tab:pink', 'tab:gray', 'tab:olive', 'tab:cyan'}``
-  (case-insensitive);
-* a "CN" color spec, i.e. ``'C'`` followed by a number, which is an index into
-  the default property cycle (:rc:`axes.prop_cycle`); the indexing is intended
-  to occur at rendering time, and defaults to black if the cycle does not
-  include color.
++--------------------------------------+--------------------------------------+
+| Format                               | Example                              |
++======================================+======================================+
+| RGB or RGBA (red, green, blue, alpha)| - ``(0.1, 0.2, 0.5)``                |
+| tuple of float values in a closed    | - ``(0.1, 0.2, 0.5, 0.3)``           |
+| interval [0, 1].                     |                                      |
++--------------------------------------+--------------------------------------+
+| Case-insensitive hex RGB or RGBA     | - ``'#0f0f0f'``                      |
+| string.                              | - ``'#0f0f0f80'``                    |
++--------------------------------------+--------------------------------------+
+| Case-insensitive shorthand equivalent| - ``'#abc'`` as ``'#aabbcc'``        |
+| string of RGB or RGBA from duplicated| - ``'#abcd'`` as ``'#aabbccdd'``     |
+| characters.                          |                                      |
```

Comment:
Reading this by itself, I'm not too sure what it means; I think a slight re-arrangement might work?
```
Case-insensitive shorthand RGB or RGBA string, made equivalent to above by duplicating characters.
```

Reply:
'shorthand' is really more of a modifier on 'string' or 'RGB or RGBA'; I don't like it in the latter half there. It reads strangely.